graphql/resolvers/directives.js

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs';
+import path from 'path';
+import { forEachField } from 'graphql-tools';
+import { getArgumentValues } from 'graphql/execution/values';
+import jwt from 'jsonwebtoken';
+import debug from 'debug';
+
+import { AuthorizationError } from '../errors';
+
+const log = debug('fcc:resolvers:directives');
+const certPath = process.cwd() + '/public.pem';
```

Comment:
If we expect this file to be there, we'll need to make sure it's available for CI to deploy too.

Another thought: if we pull this out of the hander, this will persist across Lambda invocations and we won't have to reload the file. Like I did for the database connection logic in https://github.com/freeCodeCamp/open-api/blob/staging/handler.js#L13

Reply:
I didn't think about deployment when making changes, I'll scratch my head some more